scripts/validate_docs_snippet_line_numbers.py

```diff
@@ -0,0 +1,202 @@
+import glob
+import logging
+import os
+import re
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import Dict, List, Tuple
+
+from great_expectations.data_context.util import file_relative_path
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
+
+
+OPENING_TAG: str = "<snippet>"
+CLOSING_TAG: str = "</snippet>"
+DOCUSAURUS_REGEX: re.Pattern = re.compile(r"```python file=(.+)#L(\d*)(?:-L)?(\d*)")
```

Comment:
I felt inspired by good ol HTML but I'm more than happy to change these - we can do a simple find and replace to maintain the same behavior just let me know!

Reply:
Oooh I love it!